src/patterns/filtering/multi-selection.stories.ts

```diff
@@ -0,0 +1,114 @@
+import { Component } from "@angular/core";
+import { storiesOf, moduleMetadata } from "@storybook/angular";
+import { withKnobs } from "@storybook/addon-knobs/angular";
+import { TableModule, TableModel, TableHeaderItem, TableItem } from "../../table/table.module";
+import { DropdownModule } from "../../dropdown/dropdown.module";
+
+@Component({
+    selector: "app-sample-multi-selection",
+    template: `
+    <ibm-table-container>
+        <p ibmText style="float: left; margin-right: 10px; line-height: 30px">Filter by: </p>
```

Comment:
We shouldn't be positioning things with floats ... there should be a better way to do this